### PR TITLE
10.0 mig save translation file

### PR DESCRIPTION
**Pull request by mourad-ehm**

### CODE REVIEW

File: save_translation_file/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "version": "10.0.1.0.0",
+    "category": "Tools",
+    "website": "https://www.akretion.com/",
+    "author": "Akretion, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": [
+        "base",
+    ],
+    "data": [
+        "save_translation_file_view.xml",
```
Comment (dreispt) [inline save_translation_file/__manifest__.py]: views\save_translation_file_view.xml

Comment (mourad-ehm) [reply save_translation_file/__manifest__.py]: Hi @dreispt, thanks for review.

File: save_translation_file/README.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
+server-tools/issues/new?body=module:%20
+save_translation_file%0Aversion:%20
+10.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+Akretion (http://www.akretion.com)
```
Comment (dreispt) [inline save_translation_file/README.rst:42]: Add yourself to the contributors list

**Review by dreispt — APPROVED** [review]
LGTM

File: save_translation_file/models/save_translation_file.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+from odoo import models, api, tools
+from odoo.modules import get_module_path
+from odoo.tools.misc import get_iso_codes
+
+
+class IrModuleModule(models.Model):
+    _inherit = 'ir.module.module'
+
+    @api.one
```
Comment (Fenkiou) [inline save_translation_file/models/save_translation_file.py:15]: `@api.one` is deprecated, use `@api.multi` with `for rec in self` loop or `self.ensure_one()`.

**Pull request merged**